src/web/api/src/main/java/org/locationtech/geogig/rest/geotools/SimpleFeatureStoreRepresentation.java

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2016 Boundless and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/org/documents/edl-v10.html
+ *
+ * Contributors:
+ * Erik Merkle (Boundless) - initial implementation
+ */
+package org.locationtech.geogig.rest.geotools;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.geotools.data.simple.SimpleFeatureStore;
+import org.locationtech.geogig.api.AbstractGeoGigOp;
+import org.locationtech.geogig.geotools.plumbing.ExportOp;
+import org.locationtech.geogig.rest.AsyncCommandBinaryRepresentation;
+import org.locationtech.geogig.rest.AsyncContext.AsyncCommand;
+import org.locationtech.geogig.rest.CommandBinaryRepresentationFactory;
+import org.restlet.data.MediaType;
+
+import com.google.common.io.ByteStreams;
+
+/**
+ *
+ */
+public class SimpleFeatureStoreRepresentation extends AsyncCommandBinaryRepresentation<SimpleFeatureStore> {
+
+    protected final File binary;
+
+    public SimpleFeatureStoreRepresentation(MediaType mediaType,
+        AsyncCommand<SimpleFeatureStore> cmd, File binary) {
+        super(mediaType, cmd);
+        this.binary = binary;
+    }
+
+    @Override
+    public void write(OutputStream outputStream) throws IOException {
+        int count = 0;
+        // NOTE: This is hacky. Need a better way to wait until the export is finished before
+        // streaming the export back to the consumer.
+        while (count++ < 5 && !command.isDone()) {
```

Comment:
this is terrible. But it's a crude way of waiting for the ExportOp to finish. Once finished (if it actually finishes within 5 seconds), the next part just writes the database file out to the client stream. This code needs to change, but it works for a small test right now.